daemon/graphdriver/overlay2/overlay.go

```diff
@@ -159,6 +160,18 @@ func Init(home string, options []string, idMap idtools.IdentityMapping) (graphdr
 		return nil, overlayutils.ErrDTypeNotSupported("overlay2", backingFs)
 	}
 
+	usingMetacopy, err := doesMetacopy(testdir)
+	if err == nil {
+		if usingMetacopy {
+			logger.Debugf("overlay test mount indicated that metacopy is being used")
+		} else {
+			logger.Debugf("overlay test mount indicated that metacopy is not being used")
+		}
+	} else {
+		logger.Warnf("overlay test mount did not indicate whether or not metacopy is being used: %v", err)
+		return nil, err
+	}
```

Comment:
Okay, I'm looking at this again and thinking that we should still bubble any errors up out of `doesMetacopy()` -- specifically, any failure to Mkdir/Chmod/manipulate the filesystem in any way should be bubbled up instead of swallowed by `doesMetacopy()`. 

While yes, it is optional/informational, I don't think it makes sense to silently ignore errors that may indicate an underlying system problem (like being unable to create a temporary directory, or being unable to mount a directory).